### ISSUE TITLE
Complete function 

### ISSUE THREAD

**Issue by Ruichen1024** [Ruichen1024/BME547_Classwork#11]
https://github.com/Ruichen1024/BME547_Classwork/blob/23ada209e56a954215a7ec4209ce09dd1b6ea889/get_cord.py#L2-L12

I just created this simple piece with two function to make sure that I understand how testing function works.

Richard

**Comment by dward2** [comment]
@Ruichen1024 Since I had not yet accepted your invitation to be a collaborator, I understand that you probably were not able to add me as an assignee.  In the future, using GitHub Classroom, I will automatically be a collaborator.  so, make sure to include me as an assignee on any issue you want me to see.  Thanks.

**Comment by dward2** [comment]
@Ruichen1024 Otherwise, issues look good.  Thanks.

**Comment by Ruichen1024** [comment]
> @Ruichen1024 Since I had not yet accepted your invitation to be a collaborator, I understand that you probably were not able to add me as an assignee. In the future, using GitHub Classroom, I will automatically be a collaborator. so, make sure to include me as an assignee on any issue you want me to see. Thanks.

Thanks Professor, so does it mean that I should consider using Github Classroom instead?

**Comment by dward2** [comment]
No.  I simply wanted to explain why you probably were not able to add me as an assignee.  From my perspective, you are done with this exercise.  Thanks.
